<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>497</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>20</width>
    <height>20</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <widget class="QComboBox" name="cb_com">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_bund">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_max">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>max_speed 3000</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_min">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>min_speed 20</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_speed">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>360</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DigifaceWide</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>500</string>
   </property>
   <property name="maxLength">
    <number>3276</number>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>速度设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_opencom">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>77</y>
     <width>140</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="ln_speed">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
   <property name="intValue" stdset="0">
    <number>101</number>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_setspeed">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>360</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>设置速度</string>
   </property>
   <property name="shortcut">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>70</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>端口号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>70</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>波特率：</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>500</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>100</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>速度显示值：</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>330</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>500</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>20</width>
     <height>371</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>速度显示</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>通讯设置</string>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>500</width>
     <height>16</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>500</width>
     <height>16</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>电机控制程序（IS620P）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>440</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>公司：长沙南车电气设备有限公司  作者：redmorningcn</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
